apps/prairielearn/src/pages/instructorAssessmentAccess/instructorAssessmentAccess.ejs

```diff
@@ -1,90 +0,0 @@
-            <thead>
-              <tr>
-                <th>Mode</th>
-                <th>UIDs</th>
-                <th>Start date</th>
-                <th>End date</th>
-                <th>Active</th>
-                <th>Credit</th>
-                <th>Time limit</th>
-                <th>Password</th>
-                <th>CBTF exam</th>
-                <th>PrairieTest</th>
-              </tr>
-            </thead>
-            <tbody>
-              <% access_rules.forEach(function(access_rule) { %>
-              <tr>
-                <td><%= access_rule.mode %></td>
-                <td>
-                  <%
-                    // Only users with permission to view student data are allowed
-                    // to see the list of uids associated with an access rule. Note,
-                    // however, that any user with permission to view course code
-                    // (or with access to the course git repository) will be able to
-                    // see the list of uids, because these access rules are defined
-                    // in course code. This should be changed in future, to protect
-                    // student data. See https://github.com/PrairieLearn/PrairieLearn/issues/3342
-
-                    if ((access_rule.uids == '—') || authz_data.has_course_instance_permission_view) { %>
-                    <%= access_rule.uids %>
-                  <% } else { %>
-                    <a role="button" class="btn btn-xs btn-warning" tabindex="0" data-toggle="popover" data-trigger="focus" data-container="body" data-placement="auto" title="Hidden UIDs" data-content="This access rule is specific to individual students. You need permission to view student data in order to see which ones.">Hidden</a>
-                  <% } %>
-                </td>
-                <td><%= access_rule.start_date %></td>
-                <td><%= access_rule.end_date %></td>
-                <td><%= access_rule.active %></td>
-                <td><%= access_rule.credit %></td>
-                <td><%= access_rule.time_limit %></td>
-                <td><%= access_rule.password %></td>
-                <td><%- access_rule.exam %></td>
-                <td>
-                  <% if (access_rule.pt_exam_name) { %>
-                  <a href="<%= config.ptHost %>/pt/course/<%= access_rule.pt_course_id %>/staff/exam/<%= access_rule.pt_exam_id %>"><%= access_rule.pt_course_name %>: <%= access_rule.pt_exam_name %></a>
-                  <% } else if (access_rule.exam_uuid && !access_rule.ps_exam_id) { %>
-                  <% if (devMode) { %>
-                  <%= access_rule.exam_uuid %>
-                  <% } else { %> <%# devMode %>
-                  <span class="text-danger">Exam not found: <%= access_rule.exam_uuid %></span>
-                  <% } %> <%# devMode %>
-                  <% } else { %>
-                  &mdash;
-                  <% } %>
-                  <%- access_rule.pt_exam %>
```

Comment:
@mwest1066 Do you happen to remember what was supposed to be referenced here? You were just last one to edit the code here so thought I would ask. It looks like we have `exam`, `pt_exam_id`, or `pt_exam_name` accessible here, just not a value for `pt_exam`. Thanks!